### PR TITLE
[Frontend_v2] Added option to set submission visibility on Submit page 

### PR DESCRIPTION
**Pull request by Alabhya268**
Fixed issue - #3217

visibility

Alignment and padding issue will be fixed automatically when #3220 is merged.

### CODE REVIEW

**Comment by Alabhya268** [comment]
@Ram81 @RishabhJain2018 Please review this PR , If nothing needs to be changes please merge this PR, : )

File: frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.html [hunk 1]
```diff
@@ -236,6 +236,40 @@ <h5 class="fw-light">Make Submission</h5>
...
                   </div>
                 </div>
+                <div ng-show="challenge.currentPhaseLeaderboardPublic">
+                  <p class="fs-14 w-400" ng>Select submission visibility:</p>
+                  <li>
+                      <input 
+                        type="radio" 
+                        name="submissionVisibility" 
+                        class="with-gap selectPhase" 
+                        id="isPublicubmission"
```
Comment (Ram81) [inline frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.html:246]: typo `isPublicubmission` change to `isPublicSubmission`

File: frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.html [hunk 2]
```diff
@@ -236,6 +236,40 @@ <h5 class="fw-light">Make Submission</h5>
...
+                        value="isPublicubmission"
```
Comment (Ram81) [inline frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.html:247]: we don't need value when we are using label to show the option value right? Can we remove this

File: frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.html [hunk 3]
```diff
@@ -236,6 +236,40 @@ <h5 class="fw-light">Make Submission</h5>
...
+                        ng-click="challenge.isPublicSubmission = true"
+                        />
+                      <label for="isPublicubmission" class="radio-button-text pointer">
```
Comment (Ram81) [inline frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.html:250]: same typo here

**Review by Ram81 — CHANGES_REQUESTED** [review]
@Alabhya268 I tried making a private submission using this PR and it didn't work. Can you please fix it.

File: frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.html (hunk 3 above)
Comment (Alabhya268) [reply frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.html:250]: I'll fix the typos

**Comment by Alabhya268** [comment]
> @Alabhya268 I tried making a private submission using this PR and it didn't work. Can you please fix it.

Interesting, I'll try to fix it .

**Comment by Alabhya268** [comment]
@Ram81, I've now fixed the submission visibility. Please have a look at this PR again.

**Comment by Alabhya268** [comment]
@Ram81, I've now fixed the submission visibility. Please have a look at this PR again.

**Comment by Alabhya268** [comment]
@Ram81, Submission visibility is now working properly , Could you please review this PR again?

**Comment by Alabhya268** [comment]
@Ram81 , Ping for this PR.

**Comment by Alabhya268** [comment]
@Ram81 , Please review this PR again.

**Review by Ram81 — APPROVED** [review]
LGTM 👍

**Pull request merged**